composer.json

```diff
@@ -15,13 +15,18 @@
 		"php": ">=5.6"
 	},
 	"require-dev": {
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.6",
```

Comment:
You seem to be missing `wp-coding-standards/wpcs`.

I realize it's pulled in via `WPThemeReview` too, but it is also an explicit dependency of your toolset as you include the `WordPress` ruleset in your `phpcs.xml` file.

Reply:
Thank you!